packages/protocol/contracts/governance/Election.sol

```diff
@@ -845,4 +885,40 @@ contract Election is
     }
     return res;
   }
+
+  struct SlashVotesInfo {
+    address[] groups;
+    uint256 remainingValue;
+  }
```

Comment:
Struct is used as a solution to avoid `CompilerError: Stack too deep, try removing local variables.` since I need at least 1 local variable (`remainingValue`)